pydsa/bsf.py

```diff
@@ -0,0 +1,48 @@
+"""
+
+	Breadth-first search (BFS) is an algorithm 
+	for traversing or searching tree or graph 
+	data structures. It starts at the tree root 
+	(or some arbitrary node of a graph, sometimes 
+	referred to as a 'search key'[1]) and explores 
+	the neighbor nodes first, before moving to 
+	the next level neighbors.
+
+	To find the Breadth First Search traversal
+	order of a graph given in the form of 
+	Adjacency Lists.
+	Graph is traversed from the vertex k.
+
+"""
```

Comment:
You can try this also: https://github.com/pydsa/pydsa/pull/21#discussion_r54859947